jerry-core/jerry.c

```diff
@@ -1748,6 +1748,34 @@ typedef struct
 } snapshot_globals_t;
 
 /**
+ * Write data to a specified buffer.
```

Comment:
I would use "Write data into the specified buffer" or "Write data into a buffer". I know this is copied, but it sounds better to me.